### PR TITLE
Clean env before test and after test

### PR DESCRIPTION
**Pull request by zhangyihui1**

### CODE REVIEW

**Comment by hankzh89** [comment]
Why #1 is merged in this PR?

File: test/scripts/run_test.py [hunk 1]
```diff
@@ -176,14 +179,19 @@ def recover_deps():
...
     if not validate_caselist(args.caselist):
         sys.exit(1)
 
     # generate sdk libraries.
-    if args.build:
-        build_libs(args.dependencies_dir)
+    if args.build and not build_libs(args.dependencies_dir):
         copy_libs()
-
+    else:
```
Comment (hankzh89) [inline test/scripts/run_test.py]: So this script requires `--build-deps` in order to run test cases?

Comment (zhangyihui1) [reply test/scripts/run_test.py]: done

File: test/scripts/run_test.py [hunk 2]
```diff
@@ -176,14 +179,20 @@ def recover_deps():
 
     args = parser.parse_args()
 
+    # clean environment before test.
+    recover_deps()
```
Comment (hankzh89) [inline test/scripts/run_test.py]: Why do we need to recover_deps here at the beginning of the test?

Comment (zhangyihui1) [reply test/scripts/run_test.py]: i think before test ,after test ,and error occurs ,need do clean

**Pull request merged**